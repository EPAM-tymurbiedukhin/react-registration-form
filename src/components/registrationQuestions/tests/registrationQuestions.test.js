import { mount } from 'enzyme';
import React from 'react';

import RegistrationQuestions from '../RegistrationQuestions';

import { stubRegistrationQuestionsProps } from './stubProps';

jest.mock('react-redux', () => ({
  useSelector: jest.fn(() => 'default value'), // defaultValue
}));

const setUp = (props) => render(<RegistrationQuestions {...props} />);

describe('<RegistrationQuestions />', () => {
  it('should render correctly', () => {
    const component = setUp(stubRegistrationQuestionsProps);
    expect(component.html()).toMatchSnapshot();
  });
});
