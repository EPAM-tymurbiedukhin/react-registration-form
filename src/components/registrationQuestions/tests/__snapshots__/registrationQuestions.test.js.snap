// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RegistrationQuestions /> should render correctly 1`] = `"<div class=\\"input-field col s12\\"><select name=\\"question_one\\"><option value disabled selected>Choose your option</option><option value=\\"Male\\" selected>Male</option><option value=\\"Female\\" selected>Female</option><option value=\\"Refused\\" selected>Refused</option></select><label>What is your gender?</label></div><div class=\\"input-field col s12\\"><select name=\\"question_two\\"><option value disabled selected>Choose your option</option><option value=\\"Morning\\" selected>Morning</option><option value=\\"Afternoon\\" selected>Afternoon</option><option value=\\"Evening\\" selected>Evening</option></select><label>When do you eat chocolate?</label></div><div class=\\"input-field col s12\\"><select name=\\"question_three\\"><option value disabled selected>Choose your option</option><option value=\\"21\\" selected>21</option><option value=\\"20\\" selected>20</option></select><label>In which century you was born?</label></div><button class=\\"waves-effect waves-light btn\\">submit</button>"`;
